srcpkgs/megazeux/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'megazeux'
+pkgname=megazeux
+version=2.91g
+revision=1
+build_style=gnu-makefile
+makedepends="SDL2-devel libvorbis-devel libpng-devel"
+short_desc="Game creation system"
+maintainer="Adrian Siekierka <kontakt@asie.pl>"
+license="GPL-2.0-or-later"
+homepage="http://www.digitalmzx.net/"
+wrksrc=mzx${version/.}
+_filename=${wrksrc}.tar.xz
+distfiles="http://vault.digitalmzx.net/download.php?latest=src&ver=${version}>${_filename}"
+checksum=fac0dfe1d0a6568e4338a6d9825db4d266033addfde50029feb68ad7bd5d7c4e
+
+do_configure() {
+	./config.sh --platform unix --enable-release \
```

Comment:
you can remove `do_configure()` and `do_build()` completely, if you set:
````
build_style=gnu-configure
configure_script="./config.sh"
configure_args="--platform unix --enable-release --bindir /usr/lib/megazeux
 --gamesdir /usr/bin --disable-updater"
````

Reply:
I'm a bit mixed about that, as then build_style gives misleading information (the configure system MegaZeux uses has nothing to do with GNU configure) but sure...

Reply:
you could also use `build_style=configure` if you prefer...
and please remove `do_build()` too.

Reply:
Done.